packages/react-composites/src/composites/CallWithChatComposite/MobilePane.tsx

```diff
@@ -50,6 +51,19 @@ export const MobilePane = (props: {
   }, [theme]);
   const strings = useCallWithChatCompositeStrings();
 
+  useEffect(() => {
+    // disable back button when boolean is true and mobile pane is not hidden
+    if (props.disableBackButton && !props.hidden) {
+      window.onpopstate = function () {
+        window.history.forward();
```

Comment:
Oh so this is allowing the back behavior to be triggered, but before it really happens it triggers a history.forward() to in essence ignore the back button. The alternative to this (to maybe avoid the disappearing&reappearing url parms) would be when the panel is opened do window.history.pushState... but that has more cleanup and maintainence of the history state to do so I like this implementation (but hard to predict if this will work with all customer websites)

Reply:
Yes, pushing state once something like:
```
  useEffect(() => {
    window.history.pushState(null, document.title, location.href);
  }, []);
```
does avoid disappearing&reappearing url parms. Thanks.

This will be more robust for Contoso when they back up as it avoids that brief loading of the previous url which could be anything given that they can embed the CallWithChatComposite on their web page.

However, I could not avoid situations in testing where I was doing multiple pushes to state even or using the forward button to mess up the state expectations. Will continue find a good solution today.